Improve MongoDB with Panache guide (#18010)
quarkusio/quarkus: docs/src/main/asciidoc/mongodb-panache.adoc

quarkus.log.category."io.quarkus.mongodb.panache.runtime".level=DEBUG
----

> == The PojoCodecProvider : easy object to BSON document conversion.

Reviewer: ```suggestion
== The PojoCodecProvider: easy object to BSON document conversion.
```

MongoDB with Panache uses the link:{mongodb-doc-root-url}/bson/pojos[PojoCodecProvider], with link:{mongodb-doc-root-url}/pojos/#pojo-support[automatic POJO support,
to automatically convert your object to a BSON document.

In case you encounter the `org.bson.codecs.configuration.CodecConfigurationException` exception, it means the codec is not able to
automatically convert your object.
This codec obeys the Java Bean standard, so it will successfully convert a POJO using public fields or getters/setters.
You can use `@BsonIgnore` to make a field, or a getter/setter, ignored by the codec.

> If you class didn't obey these rules (for example by including a method that starts with `get` but is not a setter),

Reviewer: ```suggestion
If your class doesn't obey these rules (for example by including a method that starts with `get` but is not a setter),
```
